docs/components/footer/index.hbs

```diff
@@ -0,0 +1,84 @@
+---
+title: Footer
+description: Showcases and describes the footer.
+nav: side/components/footer
+order: 1
+toc:
+  - title: Full footer
+    slug: full
+  - title: Langues top footer
+    slug: languages-top
+  - title: Top only footer
+    slug: top-only
+  - title: Light footer - Languages
+    slug: footer-light-lang
+  - title: Light footer - Social
+    slug: footer-light-social
+  - title: Light footer - Languages & Social
+    slug: footer-light-lang-social
+
+---
+
+{{#> page}}
+  {{#*inline "content"}}
+    <h1 class="h2">{{title}}</h1>
+    <p class="lead">
+      This page describes the footer.
+    </p>
+    <p>
+      More footer info.
+    </p>
+    {{> table-of-contents data=toc}}
+
+    <h2 class="h3" id="full">Full footer</h2>
+    <p>
+      Info about the full footer
+    </p>
+
+    {{> example path="/components/footer/demos/footer-full.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-full"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+
+    <h2 class="h3" id="languages-top">Languages top footer</h2>
+    <p>
+      Info about the languages top footer
+    </p>
+
+    {{> example path="/components/footer/demos/footer-lang-top.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-lang-top"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+    <h2 class="h3" id="top-only">Top only footer</h2>
+    <p>
+      Info about the top only footer
+    </p>
+
+    {{> example path="/components/footer/demos/footer-top-only.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-top-only"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+
+    <h2 class="h3" id="footer-light-lang">Light footer - Language</h2>
+    <p>
+      Info about the light footer with languages module
+    </p>
+
+    {{> example path="/components/footer/demos/footer-light-lang.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-light-lang"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+    <h2 class="h3" id="footer-light-social">Light footer - Social</h2>
+    <p>
+      Info about the light footer with social module
+    </p>
+
+    {{> example path="/components/footer/demos/footer-light-social.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-light-social"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+    <h2 class="h3" id="footer-light-lang-social">Light footer - Languages & Social</h2>
+    <p>
+      Info about the light footer with social module
+    </p>
+
+    {{> example path="/components/footer/demos/footer-light-lang-social.html"}}
+    {{#code-snippet "html"}}{{#get snippets "/components/footer/snippets/footer-light-lang-social"}}{{{.}}}{{/get}}{{/code-snippet}}
+
+  {{/inline}}
+{{/page}}
```

Comment:
It would be great to have a similar documentation approach on the footer page as we have on the header page.

* Tell shortly what the footer is supposed to do and why it's important
* Show a full blown example
* List all the possible elements that can go into the footer